src/storage/src/hummock/compactor.rs

```diff
@@ -145,17 +145,41 @@ impl Compactor {
         let vnode2unit: Arc<HashMap<u32, Vec<u32>>> = Arc::new(HashMap::new());
 
         for (split_index, _) in compact_task.splits.iter().enumerate() {
-            let compactor = compactor.clone();
             let iter = {
                 let iters = buffers.iter().map(|m| {
                     Box::new(m.clone().into_forward_iter()) as BoxedForwardHummockIterator
                 });
                 MergeIterator::new(iters, stats.clone())
             };
             let vnode2unit = vnode2unit.clone();
+            let compactor = compactor.clone();
             compaction_futures.push(tokio::spawn(async move {
+                let builder = GroupedSstableBuilder::new(
```

Comment:
Why add `GroupedSstableBuilder` in this PR? It seems not belong to this PR

Reply:
The latest main branch has already used this `GroupedSstableBuilder `. It's simply a refactor.

Reply:
ok